obspy/io/rg16/__init__.py

```diff
@@ -29,34 +35,41 @@
 
 .. figure:: /_images/rg16_node_orientation.png
 
-Reading
+Reading the waveforms
 -------
 The rg16 format can be read using two methods:
 
 1. Using the standard :func:`~obspy.core.stream.read` function. Optionally,
-   the format parameter can be specified as "rg16" for a modest speed-up.
+   the format parameter can be specified as "rg16" for a speed-up.
 
 2. Using the :mod:`obspy.io.rg16` specific function
    :func:`obspy.io.rg16.core._read_rg16`.
 
 Noteworthy parameters of  :func:`obspy.io.rg16.core._read_rg16`,
 which can also be passed as kwargs to :func:`~obspy.core.stream.read`:
 
-* merge: If `True` will merge traces belonging to the same channel
+* `starttime` and `endtime`: :class:`~obspy.core.utcdatetime.UTCDateTime`
```

Comment:
Monospace actually needs double backticks.. I see that a lot of the missed monospacing was already in here before this PR, but would you mind fixing it now, nevertheless, while you're at it?  :-)